docs/en/setup/Configuration-File.md

```diff
@@ -161,6 +161,7 @@ verify:
     interval: 10s   # the interval between two attempts, e.g. 10s, 1m.
   fail-fast: true  # when a case fails, whether to stop verifying other cases. This property defaults to true.
   concurrency: false # whether to verify cases concurrently. This property defaults to false.
+  batchOutput: true # whether to output the result of verify cases in batch. This property defaults to true.
```

Comment:
Hi, I'd move this to command line options, not in configuration file, otherwise we have to modify the configuration file when running locally and revert back when submit the codes into codebase.